tests/ledctl/ledctl_cmd.py

```diff
@@ -52,11 +52,20 @@ def run_ledctl_cmd(self, params: list, output=False, check=True):
         params.insert(1, self.bin)
 
         LOGGER.debug(f"Command: {params}")
-        return subprocess.run(params, capture_output=output, check=check)
-
-    def run_ledctl_cmd_decode(self, params: list):
+        print(params)
+        # Without "shell=True" returncode and some output lines are omitted
+        return subprocess.run(" ".join(params),
+                              shell=True,
+                              universal_newlines=True,
+                              capture_output=output)
+
+    # Run ledctl command and expect it to succeed
+    def run_ledctl_cmd_valid(self, params: list):
         result = self.run_ledctl_cmd(params, output=True)
-        out = result.stdout.decode("utf-8")
+        if result.returncode != 0:
+            raise Exception(
+                "Command expected to succeed, but non 0 status returned!")
+        out = result.stdout
```

Comment:
Could you please try to add verification for no errors here? I think that it should pass now and that would be beneficial to check that we are not printing error messages.

Reply:
As for now this is outside of this change's scope, due to following messages being treated as stderrs.
> /dev/shm/ledmon.conf: does not exist, using global config file
> /etc/ledmon.conf: does not exist, using built-in defaults

The change would require large rework to handle those outputs.
We discussed it offline, and agreed this is something to add afterwards.